eemeter/modeling/models/billing_caltrack.py

```diff
@@ -0,0 +1,308 @@
+            self.formula = 'upd ~ HDD'
+            self.result = hdd_res
+        elif cdd_qualified and cdd_rsquared > \
+          max([int(full_qualified)*full_rsquared,int(hdd_qualified)*hdd_rsquared,int(int_qualified)*int_rsquared]):
+            self.formula = 'upd ~ CDD'
+            self.result = cdd_res
+        else:
+            self.formula = 'upd ~ 1'
+            self.result = int_res
+        print self.formula, self.result
+    
+        if self.result is None: return None
+
+        estimated = self.result.predict(model_data)
+        estimated = pd.Series(estimated, index=model_data.index)
+        for i in range(len(estimated.index)):
+            estimated[i] = estimated[i] * \
+                calendar.monthrange(estimated.index[i].year,estimated.index[i].month)[1]
+
+        y,X = patsy.dmatrices(self.formula, model_data, return_type='dataframe')
+        self.X = X
+        for i in range(len(y.index)):
+            y['upd'][i] = y['upd'][i] * calendar.monthrange(estimated.index[i].year,estimated.index[i].month)[1]
+        self.y = y
+        self.estimated = estimated
+
+        r2 = self.result.rsquared
+        rmse = ((y.values.ravel() - estimated.values.ravel())**2).mean()**.5
+
+        if y.mean != 0:
+            cvrmse = rmse / float(y.values.ravel().mean())
+        else:
+            cvrmse = np.nan
+
+        self.r2 = r2
+        self.rmse = rmse
+        self.cvrmse = cvrmse
+
+        # For justification of these 95% confidence intervals, based on rmse,
+        # see http://stats.stackexchange.com/questions/78079/
+        #     confidence-interval-of-rmse
+        #
+        # > Let xi be your true value for the ith data point and xhat_i the
+        # >   estimated value.
+        # > If we assume that the differences between the estimated and
+        # > true values have
+        # >
+        # > 1. mean zero (i.e. the xhat_i are distributed around xi)
+        # > 2. follow a Normal distribution
+        # > 3. and all have the same standard deviation sigma
+        # > then you really want a confidence interval for sigma
+        # > ...
+        #
+        # We might decide these assumptions don't hold.
+
+        n = self.estimated.shape[0]
+
+        c1, c2 = chi2.ppf([0.025, 1-0.025], n)
+        self.lower = np.sqrt(n/c2) * self.rmse
+        self.upper = np.sqrt(n/c1) * self.rmse
+        self.n = n
+
+        self.plot()
+
+        self.params = {
+            "coefficients": self.result.params,
+            "intercept": self.result.params['Intercept'],
+            "X_design_info": X,
+            "formula": self.formula,
+        }
+
+        output = {
+            "r2": self.r2,
+            "model_params": self.params,
+            "rmse": self.rmse,
+            "cvrmse": self.cvrmse,
+            "upper": self.upper,
+            "lower": self.lower,
+            "n": self.n
+        }
+        return output
+
+    def predict(self, demand_fixture_data, params=None):
```

Comment:
This needs to conform to the interface `summed` interface.

https://github.com/openeemeter/eemeter/blob/783813c1596f6b7d248f7db1328797cbddf8524e/eemeter/modeling/models/elastic_net_base.py#L206